docs/src/main/sphinx/release/release-422.md

```diff
@@ -0,0 +1,49 @@
+# Release 422 (12 Jul 2023)
+
+## General
+
+* Add support for adding nested fields with an `ADD COLUMN` statement. ({issue}`16248`)
+* Improve performance of `INSERT` and `CREATE TABLE AS ... SELECT` queries. ({issue}`18005`)
+* Fix queries hanging when the `task.retry-policy` configuration property is set
+  to `TASK` and other worker node failures occur. ({issue}`18175 `)
+
+## Security
+
+* Add OAuth 2.0 support for validating JWT types. ({issue}`17640`)
+* Fix error when the `http-server.authentication.type` configuration property
+  is set to `oauth2` or `jwt` and the `principal-field` property has a different
+  value. ({issue}`18210`)
+
+## BigQuery connector
+
+* Add support for writing to columns with a `timestamp with time zone` type. ({issue}`17793`)
+
+## Delta Lake connector
+
+* Add support for renaming columns. ({issue}`15821`)
+* Improve performance of reading from tables with a large number of
+  [checkpoints](https://docs.delta.io/latest/delta-batch.html#-data-retention). ({issue}`17405`)
+* Disallow using the `vacuum` procedure when the max writer version is above 5. ({issue}`18095`)
+
+## Hive connector
+
+* Add support for reading `timestamp with local time zone` types. ({issue}`1240`)
+* Add a native Avro file format writer. This can be disabled with the
+  `avro.native-writer.enabled` configuration property or the
+  `avro_native_writer_enabled` session property. ({issue}`18064`)
+* Fix query failure when the `hive.recursive-directors` configuration property
+  is set to true and partition names contain non-alphanumeric characters. ({issue}`18167`)
+
+## Iceberg connector
+
+* Add support for adding nested fields with an `ADD COLUMN` statement. ({issue}`16248`)
+* Change the default file format to Parquet. The `iceberg.file-format`
```

Comment:
Wow .. do we need to call out any gotchas of that? People will suddenly have lakehouses with two different file formats.. is that ok @martint and @findepi ? 

Are there any negative side effects potentially? Performance?

Reply:
cc @raunaqmorarka

Reply:
File format is also a table property, so this change won't change anything for existing tables. It will apply only to new tables and only for ppl who haven't already explicitly specified a different config.
I don't see a problem with some tables using orc and others parquet. The users who rely on some file format specific thing (e.g. orc bloom filters) would already be specifying file format in table property or catalog configs.

Reply:
Sounds good. Lets hope we dont cause any headaches.